src/components/frame2.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import styles from "./styles/frame2.module.css";
+
+export default function Frame2({
+  toggleFrame,
+  addMember,
+}: {
+  toggleFrame: (x: number) => void;
+  addMember: (name: string, desc: string) => void;
+}) {
+  const [name, setName] = useState("");
+  const [desc, setDesc] = useState("");
+
+  const save = () => {
+    addMember(name, desc);
+    toggleFrame(0);
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.inputWrapper}>
+        <input
+          className={styles.input}
+          type="text"
+          placeholder="이름을 입력하세요"
+          value={name}
+          onChange={(e) => setName(e.target.value)}
+        />
+      </div>
+      <div className={styles.inputWrapper}>
+        <input
+          className={styles.input}
+          type="text"
+          placeholder="설명을 입력하세요"
+          value={desc}
+          onChange={(e) => setDesc(e.target.value)}
+        />
+      </div>
+      <div className={styles.buttonWrapper}>
+        <button className={styles.saveButton} onClick={() => save()}>
```

Comment:
```suggestion
        <button className={styles.saveButton} onClick={save} disabled={ture}>
```
disabled 속성을 주어 버튼을 눌리지 않게 할 수 있습니다. css에서도 `:disabled`로 스타일링 할 수 있고요!

또 save 같은 인자를 받지 않는 함수의 경우 이렇게 단순화해서 적을 수도 있습니다. ClickEvent 인스턴스가 인수로 넘어가긴 하는데, 어차피 인자를 받지 않는 함수여서 무시됩니다.